data/transport missions.txt

```diff
@@ -1868,6 +1872,309 @@ mission "Terraforming 5"
+
+
+
+mission "terraforming timer"
+	landing
+	invisible
+	to offer
+		has "Terraforming 5: done"
+	on offer
+		event "terraforming timer" 4 8
+		fail
+
+event "terraforming timer"
+
+mission "Terraforming 6"
+	landing
+	name "Unlawful Use of an Asteroid"
+	description `Travel to <destination> by <date> for questioning on your "unlawful use of an asteroid."`
+	source
+		government "Republic"
+		not attributes "deep"
+	destination "Earth"
+	deadline
+	to offer
+		has "event: terraforming timer"
+		or
+			has "main plot completed"
+			and
+				not "event: pug invasion"
+				not "event: fwc pug invasion"
+	
+	on offer
+		conversation
+			`When you land on <origin> and exit your ship, you are stopped by a Navy officer with a group of guards. "Captain <last>. You are wanted for..." the officer pauses and looks down at the tablet in his hand. "The 'unlawful use of an asteroid.' You are to travel to <destination> immediately for questioning."`
+			choice
+				`	"Understood, officer."`
+					accept
+				`	"I'm wanted for what?"`
+				`	(Run away.)`
+					flee
+			
+			`	"The unlawful use of an asteroid, Captain. I'm not entirely sure what that means, but I've been told to send you to <planet>. Your compliance would be much appreciated."`
+			choice
+				`	"Alright. Hopefully I can get an answers as to what this is about."`
+					accept
+				`	(Run away.)`
+					flee
+			
+	on decline
+		"reputation: Republic" <?= -10
+	on fail
+		"reputation: Republic" <?= -10
+	on complete
+		conversation
+			`You are quickly escorted from the crowded <planet> spaceport to a court building in a wealthy district of the city and asked multiple questions pertaining to when you crashed an asteroid into Rand. "Why did you purposefully do crash an asteroid into a planet? What did you expect to happen? Do you know how many people could have died?" You answer the questions to the best of your ability, and after roughly an hour of questioning you are transferred to a different room where you find that Eric, Alaric, and Amy are all here as well.`
+			choice
+				`	"What are we going to do?"`
+					goto lawyer
+				`	"Do you know what we're doing here?"`
+			
+			`	"Apparently someone had a problem with us terraforming Rand," Amy says, looking rather disappointed.`
+			
+			label lawyer
+			`	"Don't worry," Alaric says. "Our lawyer is on his way right now. We'll all be out of here by the end of the day."`
+			`	Alaric's optimism turns out to be justified, as the lawyer quickly finds that there are no actual laws on the books that define lawful or unlawful uses of an asteroid. Since no one was hurt by the asteroid impact and data that Amy had been collecting shows that Rand's climate has changed for the better so far, the case is quickly dropped and you are escorted back to your ship.`
```

Comment:
I do find the lack of laws unlikely - in 1000 years there's certainly going to be some renegade corporation pilot who gets mad for being shorted a couple hundred credits and 'accidentally' lets loose his payload into a station or refinery.

Perhaps there's a local ordinance in some other system that doesn't apply to Rand, or, better yet, an obscure one that does 👍 

At this point what's the minimum time that has elapsed since the initial impact? Climate science isn't exactly a rapid thing - rapid climate changes are generally ELE's in the eyes of today's public. Of course, handwave `"it's the future, man"` means this is likely low-priority.

Reply:
/shrug